src/Oscoin/Crypto/Hash.hs

```diff
@@ -92,7 +108,12 @@ class
     -- | The zero hash. Also the minimum hash value.
     zeroHash  :: Hash crypto
 
-    shortHash :: Hash crypto -> ByteString
+    -- | A short hash, in the similar spirit of the BTC one. This is typically
+    -- ripemd160(some_other_hash).
```

Comment:
:thinking: - what do you mean by "some other hash"?

Reply:
Good question: this is essentially predated from BTC, see [here](https://en.bitcoin.it/wiki/Base58Check_encoding). In the section "Encoding a Bitcoin address" it reads:

```
Pay-to-pubkey-hash (p2pkh): payload is RIPEMD160(SHA256(ECDSA_publicKey)) where ECDSA_publicKey is a public key the wallet knows the private key for; version 0x00 (these addresses begin with the digit '1')
``` 

Apparently RIPEMD160 is very well suited for these things due to its "avalanche effect". However, we are not doing it 100% BTC-compliant , as we don't use SHA_256 but Blake2b_256.

This is what I meant with "some other hash" 😁

Reply:
I think that in the comment, I should have said "This is typically RIPMD160(HashAlgorithm crypto)" or something like that 😅

Reply:
Ohhhhhhh. Maybe yeah even `eg. ripemd160(blake2b(x))`

Reply:
35cf933